packages/macros/tests/babel/dependency-satisfies.test.ts

```diff
@@ -134,13 +134,39 @@ describe(`dependencySatisfies`, function () {
         expect(runDefault(code)).toBe(true);
       });
 
+      test(`it considers a project's used pre-releases as allowed`, () => {
+        project.addDependency('foo', '4.2.0-beta.1');
+        let code = transform(
+          `
+          import { dependencySatisfies } from '@embroider/macros';
+          export default function() {
+            return dependencySatisfies('foo', '>= 3.27.0-alpha.1 || >= 3.27.0-beta.1');
+          }
+        `
+        );
+        expect(runDefault(code)).toBe(true);
+      });
+
+      test(`in monorepos, the * dep is commonly used to mean "latest"`, () => {
+        project.addDependency('foo', '*');
+        let code = transform(
+          `
+          import { dependencySatisfies } from '@embroider/macros';
+          export default function() {
+            return dependencySatisfies('foo', '>= 3.27.0-alpha.1 || >= 3.27.0-beta.1');
```

Comment:
the star shoudl go here:

```suggestion
            return dependencySatisfies('foo', '*');
```

Reply:
Not for what I'm wanting to test tho. 
Example: https://github.com/embroider-build/embroider/blob/main/packages/util/addon/ember-private-api.js#L11
(and then my consuming project has `*`)
(unless I totally misunderstand this test's APIs (project.addDependency, etc) -- possible!)

Reply:
I think I added the dependency correctly?

Reply:
Thanks for updating the tests!!